ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml

```diff
@@ -411,12 +411,36 @@ spec:
         - name: credentials
           workspace: registry-credentials
 
+    - name: validate-catalog-format
+      taskRef:
+        name: validate-catalog-format
+        kind: Task
+      runAfter:
+        # This is just a temporary location until we have a proper
+        # FCB branch based on config
+        - build-fragment-images
+      when:
+        - input: "$(tasks.detect-changes.results.affected_catalogs)"
+          operator: notin
+          values: [""]
+      params:
+        - name: pipeline_image
+          value: "$(params.pipeline_image)"
+        - name: affected_catalogs
+          value: "$(tasks.detect-changes.results.affected_catalogs)"
+        - name: operator_path
+          value: "$(tasks.detect-changes.results.operator_path)"
+      workspaces:
+        - name: source
+          workspace: repository
+          subPath: src
```

Comment:
We need to make sure the makefile cannot tamper with the files in the repository workspace.